### ISSUE TITLE
Question: how to determine if Open() is successful?

### ISSUE THREAD

**Issue by wjquigs11** [ttlappalainen/NMEA2000#418]
Is there any way to determine if the N2K Open succeeded? It returns "true" even if the interface is not physically connected to a bus.
For now I'm using packet receive count == 0 after some time, but that seems unreliable.

**Comment by unxs0** [comment]
@wjquigs11  A loopback type check. This would require a hardware standard that may exist already.

**Comment by wjquigs11** [comment]
No, I meant a check for physical connection to the bus. I came across a better solution than mine, which is a bool N2Kopen = false, changed to true in the message handler (i.e., will only be true if we've received a valid packet to process).
